app/views/services/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+
+  <h1>Services Index</h1>
```

Comment:
Actually h1 is Craftpeople

![image](https://user-images.githubusercontent.com/108366278/202849750-536383d1-0ea5-41c1-a6ac-b1c48698bb2a.png)

We will create a partial with titles inside, you'll just have to copy paste the relevant one in the index. For now, h1 craftpeople is enough, to avoid confusion